examples/host-vars.yml

```diff
@@ -24,4 +24,11 @@ matrix_coturn_turn_static_auth_secret: ""
 
 # A secret used to protect access keys issued by the server.
 # You can put any string here, but generating a strong one is preferred (e.g. `pwgen -s 64 1`).
-matrix_synapse_macaroon_secret_key: ""
\ No newline at end of file
+matrix_synapse_macaroon_secret_key: ""
+
+# Mautrix telegram
```

Comment:
My goal was to have the `examples/host-vars.yml` file contain only the absolutely required configuration for having the playbook run.

Most other additional services and extensions require additional configuration, which can be discovered from the role's `defaults/main.yml` file or, even better, from the the docs.

Would you get rid of these here and create some new documentation file for this bridge integration in `docs/` (possibly `docs/configuring-playbook-bridge-mautrix-telegram.md`?
We can then link to that file from within `docs/configuring-playbook.md`.

Reply:
I'll write some doc :)

Reply:
Don't hesitate to tell me, if it is too light